applications/nrf_desktop/configuration/nrf52840dk_nrf52840/pm_static_ZDebugB0.yml

```diff
@@ -1,11 +1,11 @@
-b0:
+b0_provision:
```

Comment:
isn't the new naming somehow odd? Container is named after one of the elements.
b0_provision contains b0 and provision

Reply:
@jonathannilsen can probably better speak to the reasoning behind the name here. AFAIK, the name of this container isn't very important so long as it doesn't match the name of the child image to which it belongs (NCSDK-7982).

Since `b0_provision` is just a container partition, perhaps `b0_span` is clearer for its purpose in the configuration?

Reply:
I named it `b0_provision` since that seemed to fit with the naming pattern used for other container partitions in samples/bootloader/pm.yml. I agree that a name like `b0_span` is clearer.